amazon-s3/5.6/modules/ROOT/pages/amazon-s3-connector-config-topics.adoc

```diff
@@ -9,6 +9,151 @@ In the `Create Object` operation, set the `Content Length` to a value greater th
 
 To encrypt objects that you want to store in S3 buckets using customer managed master keys, specify a Customer Master Key ID in the `KMS Master Key` field in the `Create Object` configuration.
 
+== Prerequisites for Source Operations
```

Comment:
Should this be moved to the beginning of the queue created by source operations topic?
During my first reading, I thought that this topic set was arranged around
1. Queue exists
2. Source operation creates queue